app/views/browse/service-instance.html

```diff
@@ -0,0 +1,102 @@
+                      project-name="{{serviceInstance.metadata.namespace}}"
+                      alerts="alerts">
+                    </delete-link>
+                  </li>
+                </ul>
+              </div>
+              {{serviceInstance | serviceInstanceDisplayName:serviceClasses}}
+              <small class="meta">created <span am-time-ago="serviceInstance.metadata.creationTimestamp"></span></small>
+            </h1>
+            <labels labels="serviceInstance.metadata.labels" clickable="true" kind="instance" project-name="{{serviceInstance.metadata.namespace}}" limit="3"></labels>
+          </div>
+        </div>
+      </div><!-- /middle-header-->
+      <div class="middle-content" persist-tab-state>
+        <div class="container-fluid">
+          <div class="row" ng-if="serviceInstance">
+            <div class="col-md-12">
+              <uib-tabset>
+                <uib-tab active="selectedTab.details">
+                  <uib-tab-heading>Details</uib-tab-heading>
+                  <div class="resource-details">
+                    <div class="row">
+                      <div class="col-lg-6">
+                        <h3>Plan</h3>
+                        <p>
+                          {{plan.description}}
+                        </p>
+                        <dl class="dl-horizontal left">
+                          <dt ng-if-start="serviceClass.description">Description:</dt>
+                          <dd ng-if-end>
+                            <p class="pre-wrap" ng-bind-html="serviceClass.description | linky"></p>
+                            <p class="pre-wrap" ng-bind-html="serviceClass.externalMetadata.longDescription | linky"></p>
+                          </dd>
+                          <dt>Status:</dt>
+                          <dd>
+                            <status-icon status="serviceInstance | serviceInstanceStatus" disable-animation></status-icon>
+                            <span flex>{{serviceInstance | serviceInstanceStatus | sentenceCase}}</span>
+                          </dd>
+                          <dt ng-if-start="serviceInstance | serviceInstanceMessage">Status Reason:</dt>
+                          <dd ng-if-end>
+                            {{serviceInstance | serviceInstanceMessage}}
+                          </dd>
+                          <!--<dd>
+                            <status-icon status="serviceInstance | serviceInstanceStatus" disable-animation></status-icon>
+                            <span flex>
+                              {{serviceInstance | serviceInstanceStatus}}<span ng-if="serviceInstance | serviceInstanceMessage">. {{serviceInstance | serviceInstanceMessage}}</span>
+                            </span>
+                          </dd>-->
+                        </dl>
+                      </div>
+                      <div class="col-lg-6">
+                        <resource-service-bindings
```

Comment:
We might rename this component to just `service-bindings` if we're using it for service instances too.

Reply:
I can do this, it's going to affect a handful of files.